server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ConnectSessionController.java

```diff
@@ -141,6 +142,21 @@ public ListResponse<CheckResult> check(@PathVariable String sessionId, @RequestB
         return Responses.list(this.sqlCheckService.check(connectionSession, req));
     }
 
+    /**
+     * 对多个数据库进行sql检查
+     * todo
+     * @param req
+     * @return
+     */
+    @ApiOperation(value = "sqlCheck", notes = "对多个sql 脚本的内容进行静态检查")
+    @PostMapping("sessions/multiple/sqlCheck")
+    public ListResponse<CheckResult> multipleCheck(@RequestBody MultipleSqlCheckReq req) {
```

Comment:
this method should thrown an exception first~

Reply:
have done